### DISCUSSION TITLE
I have  no cursorline and no shade for nvim tree

### DISCUSSION THREAD

**Discussion by ajayd-san** [olimorris/onedarkpro.nvim#193] [Q&A]
hey, 
I have the cursor line working for other themes but the cursorline does not work for onedarkpro. I've even tried setting a random color explicitly during the setup. It just doesn't work. 

![OnedarkPro](https://github.com/olimorris/onedarkpro.nvim/assets/54715852/f2d76f8b-ccc0-4c00-973f-68a958e3995d)
![Catppuccin](https://github.com/olimorris/onedarkpro.nvim/assets/54715852/37199580-d9c9-4857-83e9-1bbd537060de)

And similarly the editor and nvim tree share the same bg color, i.e nvim-tree does not have a different shade like vscode. 

![nvim tree has same bg color with onedarkpro](https://github.com/olimorris/onedarkpro.nvim/assets/54715852/dd482b9f-aed0-4eee-bcd3-08d1dc6f95e8)
![nvim tree has different bg with catppuccin](https://github.com/olimorris/onedarkpro.nvim/assets/54715852/47fd10af-11ad-4918-aacb-3148352230d5)

any help is appreciated. I'm ready to provide any other info that is needed. thanks

**Comment by olimorris** [comment]
Have you turned on `cursorline` in the theme's options?

I'd suggest reading nvim-tree's documentation for the available highlight groups which you can change to your liking. The readme for this plugin covers how you can make colors darker or lighter.

**Reply by ajayd-san** [reply]
hey, thanks for swift reply!!

I enabled cursor line in my plugin settings and it started working as intended. 

However, I'm not really sure how I can target the specific highlight group for nvim-tree after finding out its highlight group with `:Inspect`. Can you help me out here?

**Reply by olimorris** [reply]
The readme covers all of this. See configuring highlight groups 👍🏼

**Reply by olimorris** [reply]
The readme covers all of this. See configuring highlight groups 👍🏼

**Reply by olimorris** [reply]
https://github.com/olimorris/onedarkpro.nvim#configuring-highlight-groups

**Reply by olimorris** [reply]
@default-303 - I've just updated the readme. I wasn't explicit enough about creating highlight groups. Hope it helps
